services/web/server/src/simcore_service_webserver/login/plugin.py

```diff
@@ -54,7 +54,7 @@ def setup_login_storage(app: web.Application):
 def _setup_login_options(app: web.Application):
     settings: SMTPSettings = get_email_plugin_settings(app)
 
-    cfg = settings.dict(exclude_unset=True)
+    cfg = settings.dict(exclude_none=True)
```

Comment:
are you sure about this? :-)
Could you please add some test that verify your example

Reply:
I am more than sure about this. It strips away files which are set by default. The wrong keyword was used.

Reply:
Uhm.. can't reproduce in testing only when application is running. @pcrespov there is something fishy here

Reply:
It looks better not to use exclude_none. Since it can be handedly by the model below.

Reply:
Fixed this as discussed